sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -1626,7 +1632,7 @@ public virtual Response<RecognizeLinkedEntitiesResultCollection> RecognizeLinked
                 string id = item switch
                 {
                     TextDocumentInput tdi => tdi.Id,
-                    DetectLanguageInput dli => dli.Id,
+                    LanguageInput ldi => ldi.Id,
```

Comment:
nit: should it be `li` instead of `ldi`?

Reply:
Will fix in next codegen iteration